webpack.config.js

```diff
@@ -38,6 +38,8 @@ Encore
   .addEntry('case_new_prepare', './assets/case/municipality_and_board_selector.js')
   .addEntry('mail_templates_custom_data', './assets/mail_templates/custom_data.js')
   .addEntry('document_case_event', './assets/document/case_event.js')
+  .addEntry('case_event_copy_select2', './assets/case_event/copy_select2.js')
+  .addEntry('document_copy_select2', './assets/document/copy_select2.js')
```

Comment:
“_select2” should not be a part of these names. We don't want to rename if we switch to https://tom-select.js.org/, say. I recommend switching to Tom Select (which, by the way, is used by Easy Admin).

Reply:
Removed the `_select2` part of the names. 

Lets wait with the change to `tom-select` (https://symfony.com/doc/current/frontend/ux.html).